data/doxygen.h

```diff
@@ -7,45 +7,83 @@
  * sequence of steps which are executed in order or through control flow statements. The
  * behavior of each step is defined in the LUA scripting language.
  *
- * The C++ classes for modeling sequences and steps are, unsurprisingly,
- * \ref task::Sequence "Sequence" and \ref task::Step "Step". A sequence acts like a
- * container for steps and can be executed in the current thread with the member function
+ * The library provides the main modeling classes for sequences and steps, functionality
+ * for executing them in the current thread or in a parallel one, as well as serialization
```

Comment:
> executing them in the current thread or in a parallel

Oh it is? I hope we have this try-catch stuff for both scenarios. I seem to remember catch problems that showed only up after I started using the executor instead of the direct 'run this step' button in Taskomat. While developing.

Just comes to my mind right now. I thought direct execution was not 'the supported way' and I did not investigate further.

Reply:
All exception handling for the Lua engine is done in `Step::execute()`, and any exception from there is picked up by `Sequence::execute()`. It is only above that level that there may be differences. So far, I'd say direct execution is supported.